plugin/trino-druid/src/test/java/io/trino/plugin/druid/BaseDruidConnectorTest.java

```diff
@@ -302,6 +309,84 @@ public void testFilteringForTablesAndColumns()
         Assert.assertEquals(actualColumns, expectedColumns);
     }
 
+    @Test
+    public void testAggregationPushdown()
+    {
+        assertThat(query("SELECT count(*) FROM orders")).isFullyPushedDown();
+
+        // for varchar only count is pushed down
+        assertThat(query("SELECT count(comment) FROM orders")).isFullyPushedDown();
+
+        // for timestamp
+        assertThat(query("SELECT count(__time) FROM orders")).isFullyPushedDown();
+        assertThat(query("SELECT min(__time) FROM orders")).isFullyPushedDown();
+        assertThat(query("SELECT max(__time) FROM orders")).isFullyPushedDown();
+
+        // for double
+        assertThat(query("SELECT count(totalprice) FROM orders")).isFullyPushedDown();
+        assertThat(query("SELECT min(totalprice) FROM orders group by custkey")).isFullyPushedDown();
+        assertThat(query("SELECT max(totalprice) FROM orders")).isFullyPushedDown();
+        assertThat(query("SELECT sum(totalprice) FROM orders")).isFullyPushedDown();
+        assertThat(query("SELECT avg(totalprice) FROM orders")).isFullyPushedDown();
+        assertThat(query("SELECT stddev(totalprice) FROM orders")).isFullyPushedDown();
+        assertThat(query("SELECT stddev_samp(totalprice) FROM orders")).isFullyPushedDown();
+        assertThat(query("SELECT stddev_pop(totalprice) FROM orders")).isFullyPushedDown();
+        assertThat(query("SELECT variance(totalprice) FROM orders")).isFullyPushedDown();
+        assertThat(query("SELECT var_samp(totalprice) FROM orders")).isFullyPushedDown();
+        assertThat(query("SELECT var_pop(totalprice) FROM orders")).isFullyPushedDown();
+        assertThat(query("SELECT stddev(double_col) FROM singlerow")).isFullyPushedDown();
+        assertThat(query("SELECT stddev_samp(double_col) FROM singlerow")).isFullyPushedDown();
+        assertThat(query("SELECT stddev_pop(double_col) FROM singlerow")).isFullyPushedDown();
+        assertThat(query("SELECT variance(double_col) FROM singlerow")).isFullyPushedDown();
+        assertThat(query("SELECT var_samp(double_col) FROM singlerow")).isFullyPushedDown();
+        assertThat(query("SELECT var_pop(double_col) FROM singlerow")).isFullyPushedDown();
+        assertThat(query("SELECT stddev(double_col) FROM nodata")).isFullyPushedDown();
+        assertThat(query("SELECT stddev_samp(double_col) FROM nodata")).isFullyPushedDown();
+        assertThat(query("SELECT stddev_pop(double_col) FROM nodata")).isFullyPushedDown();
+        assertThat(query("SELECT variance(double_col) FROM nodata")).isFullyPushedDown();
+        assertThat(query("SELECT var_samp(double_col) FROM nodata")).isFullyPushedDown();
+        assertThat(query("SELECT var_pop(double_col) FROM nodata")).isFullyPushedDown();
```

Comment:
I like the "nodata" case ( https://github.com/trinodb/trino/pull/7948#issuecomment-844054595 )

for general test structure, let's follow the estanbilished pattern, which also helps exploit some known corner cases. See PostgreSQL tests and see https://github.com/trinodb/trino/pull/7948 for directly reusable test infra

Reply:
Can't really use the base structure as is as Druid does not support Create or insert yet if you meant to say I should use the test as is, if you mean I should add filter conditions let me know.